SABT_MainUnit/script_hindi.h

```diff
@@ -78,11 +78,9 @@ glyph_t glyphs_hindi[SCRIPT_HINDI_LENGTH];
 
 //Multi-cell - RU
 
-script_t subscript_hin_ru_1;
-glyph_t glyphs_hin_ru_1[1];
+glyph_t hin_ru_1 = {HIN_RU_1, "RU", NULL, NULL};
+glyph_t hin_ru_2 = {HIN_RU_2, "RU_2", NULL, NULL};

```

Comment:
Can we rewrite this so we don't double declare hin_ru_1 etc both here and in the glyph array?